include/CoreGraphics/CGColorSpace.h

```diff
@@ -17,14 +17,16 @@
 #pragma once
 
 #import <CoreGraphics/CoreGraphicsExport.h>
+#import <CoreGraphics/CGBase.h>
 #import <CoreGraphics/CGDataProvider.h>
 
 COREGRAPHICS_EXPORT void CGColorSpaceRelease(CGColorSpaceRef space);
 COREGRAPHICS_EXPORT CGColorSpaceRef CGColorSpaceRetain(CGColorSpaceRef space);
 COREGRAPHICS_EXPORT const CFStringRef kCGColorSpaceGenericGray;
 COREGRAPHICS_EXPORT const CFStringRef kCGColorSpaceGenericRGB;
 COREGRAPHICS_EXPORT const CFStringRef kCGColorSpaceGenericCMYK;
-;COREGRAPHICS_EXPORT const CFStringRef kCGColorSpaceGenericRGBLinear;
+;
+COREGRAPHICS_EXPORT const CFStringRef kCGColorSpaceGenericRGBLinear;
```

Comment:
It seems to have broken a line where there was an extra ;

---
In reply to: [75770228](https://github.com/Microsoft/WinObjC/pull/664#discussion_r75770228) [](ancestors = 75770228)